modules/ROOT/pages/_partials/common-write.adoc

```diff
@@ -1,12 +1,12 @@
-== Writing to Existing Files
+== Write to Existing Files
 //INCLUDED IN file-write, ftp-write, and sftp-write adoc files
 
 There are several file write modes you can use when writing content to a file:
 
 * OVERWRITE: If the file exists, overwrite it completely.
-* APPEND: If the file exists, write the content at the end of it the existing file.
-* CREATE_NEW: Create a new file. If the file already exists, then you will receive an error.
+* APPEND: If the file exists, add the content to the end of an existing file.
```

Comment:
"the content" -> "its content" ?

Reply:
Changed to:
* APPEND: If the file exists, add its content to the end of an existing file.